### ISSUE TITLE
Segmentation fault since 2.26 version

### ISSUE THREAD

**Issue by tr1ckyf0x** [yonaskolb/XcodeGen#1177]
Installed via homebrew
Version: 2.26.0
MacBook Pro 2021 M1 Max
macOS 12.2.1

```
xcodegen generate
⚙️  Generating plists...
⚙️  Generating project...
[1]    6808 segmentation fault  xcodegen generate
```

**Comment by tr1ckyf0x** [comment]
The problem looks not to be resolved after previous fix. It stopped to happen in debug build, but occurs in release.

**Comment by yonaskolb** [comment]
https://github.com/yonaskolb/XcodeGen/pull/1198 was merged and released in 2.28.0. Are the issues resolved now? Please re-open if not
